### PR TITLE
nav-redesign v2

### PR DESCRIPTION
**Pull request by rybmat**
testing if hubs in mercury navigation are visible only on english wikis

### CODE REVIEW

File: src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java [hunk 1]
```diff
@@ -153,4 +163,15 @@ public boolean isFooterVisible() {
       return false;
     }
   }
+
+  public boolean areHubLinksVisible() {
+    try {
+      return gamesHub.isDisplayed()
+          && moviesHub.isDisplayed()
+          && tvHub.isDisplayed();
+    } catch (NoSuchElementException e) {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/elements/mercury/components/Navigation.java]: MAJOR Either log or rethrow this exception. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS1166)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java [hunk 2]
```diff
@@ -73,4 +73,23 @@ public void mercury_navigation_isFooterAlwaysVisible() {
...
+    init();
+
+    topBar.openNavigation();
+    Assertion.assertTrue(navigation.areHubLinksVisible());
+    Assertion.assertTrue(navigation.isFooterVisible());
+  }
+
+  @Execute(onWikia = MercuryWikis.DE_WIKI)
+  @Test(groups = "mercury_navigation_navigationOnNonEnglishWiki")
+  public void mercury_navigation_navigationOnNonEnglishWiki() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java [hunk 3]
```diff
@@ -73,4 +73,23 @@ public void mercury_navigation_isFooterAlwaysVisible() {
     driver.executeScript("window.scrollTo(100, document.body.scrollHeight)");
     Assertion.assertTrue(navigation.isFooterVisible());
   }
+
+  @Test(groups = "mercury_navigation_navigationOnEnglishWiki")
+  public void mercury_navigation_navigationOnEnglishWiki() {
```
Comment (WikiaTech) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/NavigationTest.java]: MAJOR Rename this method name to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [rule](http://sonar-s1/coding_rules#rule_key=squid%3AS00100)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues:
* MAJOR 3 major

Watch the comments in this conversation to review them.

**Comment by nikodamn** [comment]
LGTM

**Pull request merged**